packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -614,27 +615,51 @@ class RenderListWheelViewport
   }
 
   @override
-  double getOffsetToReveal(RenderObject target, double alignment) {
-    final ListWheelParentData parentData = target.parentData;
-    final double centerPosition = parentData.offset.dy;
-
-    if (alignment < 0.5) {
-      return centerPosition + _topScrollMarginExtent * alignment * 2.0;
-    } else if (alignment > 0.5) {
-      return centerPosition - _topScrollMarginExtent * (alignment - 0.5) * 2.0;
-    } else {
-      return centerPosition;
-    }
+  RevealedOffset getOffsetToReveal(RenderObject target, double alignment, {Rect rect}) {
+    // `target` is only fully revealed when in the selected/center position. Therefore,
+    // this method always returns the offset that shows `target` in the center position,
+    // which is the same offset for all `alignment` values.
+
+    rect ??= target.paintBounds;
+
+    // The child will be the topmost object before we get to the viewport.
```

Comment:
I think this comment is more confusing than illuminating. (I thought it meant "topmost" in paint order.)